daly-bms-uart.cpp

```diff
@@ -80,6 +80,15 @@ bool Daly_BMS_UART::getPackMeasurements() // 0x90
 #ifdef DEBUG_SERIAL
         DEBUG_SERIAL.print("<DALY-BMS DEBUG> Receive failed, V, I, & SOC values won't be modified!\n");
 #endif
+    //if we got no or wrong data lets unset it
+    //need to be modified to clear the compiler warning
+
+    //new function unset all
+    //memset(&get, 0, sizeof(get));
+    //get.packVoltage = NAN;
+    //get.packCurrent = NAN;
+    //get.packSOC = NAN;
```

Comment:
that function i have added because when read arror from the bms, we get allways the last valid data, so we get cleared data when readinf fail or no connection